.github/workflows/ci-actions.yml

```diff
@@ -13,18 +13,18 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        GOVER: ['1.21.1', '1.20.8']
+        GOVER: ['stable', 'oldstable']
```

Comment:
Do we want upstream to dictate our CI? @armfazh

Reply:
The motivation for this is to use the latest Go versions supported by upstream. Go 1.20 won't receive any new updates anymore.

I understand if you want to pin the older version here, but that means frequent updates like this. It's a trade-off to be made.

Reply:
Ok. We can revert if Armando disagrees.

Reply:
I think it's ok to explicitly maintain the two most recent versions, changes happen not so often. So, let's leave it as is.

Reply:
Do you have a preference for hardcoding the major version as opposed to using `stable` and `oldstable`?

Note that specifying `1.21` will install the latest patch version: https://github.com/actions/setup-go/blob/6c1fd22b67f7a7c42ad9a45c0f4197434035e429/__tests__/setup-go.test.ts#L433-L439

In the current situation on the `main` branch we even hardcoded the patch version which I think is unnecessary.

Reply:
the style e.g. `1.21` should be enough.

Reply:
This has been addressed in #477.